source/Trashbot.js

```diff
@@ -27,7 +25,7 @@ export default class Trashbot extends Pixi.Sprite {
                     child.onCollision(this)
                 }
             } else if (child instanceof Junkership) {
-                if (Utility.hasCollision(this, child)) {
+                if (Utility.hasCollision(this, child.hitBox)) {
```

Comment:
See what good loose coupling we had here? We didn't have to change `Utility.hasCollision()` at all, only the parameters we passed it. :)